doc/jsk_topic_tools/scripts/and_node.rst

```diff
@@ -0,0 +1,36 @@
+and_node.py
+===========
+
+
+What is this?
+-------------
+
+A node that takes bool values and returns the result of the ``and`` operation.
+
+
+Subscribing Topics
+------------------
+
+* ``~input01`` (``std_msgs/Bool``)
+
+  input bool value.
+
+* ``~input02`` (``std_msgs/Bool``)
```

Comment:
Maybe we can have more number of input topics by using rosparam.

https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_tools/src/test_topic_published.py
https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_tools/test/test_topic_published.launch

Reply:
Modified